docs/operations/migration_policies.md

```diff
@@ -0,0 +1,161 @@
+# Migration Policies
+
+Migration policies provides a new way of applying migration configurations to Virtual Machines. The policies
+can refine Kubevirt CR's `MigrationConfiguration` that sets the cluster-wide migration configurations. This way,
+the cluster-wide settings serve as a default while can be changed, removed or added by the migration policy.
+
+## Overview
+
+KubeVirt supports [Live Migrations](./live_migration.md) of Virtual Machine workloads.
+Before migration policies were introduced, migration settings could be configurable only on the cluster-wide
+scope by editing [KubevirtCR's spec](https://kubevirt.io/api-reference/master/definitions.html#_v1_kubevirtspec)
+or more specifically [MigrationConfiguration](https://kubevirt.io/api-reference/master/definitions.html#_v1_migrationconfiguration)
+CRD.
+
+Several aspects (although not all) of migration behaviour that can be customized are:
+- Bandwidth
+- Auto-convergence
+- Post/Pre-copy
+- Max number of parallel migrations
+- Timeout
+
+Migration policies generalize the concept of defining migration configurations, so it would be
+possible to apply different configurations to specified groups of VMs.
+
+Such capability can be useful for a lot of different use cases on which there is a need to differentiate
+between different workloads. Differentiation of different configurations could be needed because different
+workloads are considered to be in different priorities, security segregation, workloads with different
+requirements, help to converge workloads which aren't migration-friendly, and many other reasons.
+
+## API Examples
+
+#### Migration Configurations
+Currently, MigrationPolicy's spec will only include the currently relevant configurations from KubevirtCR's
+MicrationConfiguration:
+```yaml
+apiVersion: migrations.kubevirt.io/v1alpha1
+kind: MigrationPolicy
+  spec:
+    allowAutoConverge: true
+    bandwidthPerMigration: 217Ki
+    completionTimeoutPerGiB: 23
+    allowPostCopy: false
+```
+
+In the future more configurations can be added, either being cloned from KubevirtCR's MigrationConfiguration or
+by adding a configuration that is not suited for cluster-wide context.
```

Comment:
While correc,t I'D keep this out of the user-guide.

Reply:
done :+1: